writing_content.Rmd

```diff
@@ -468,12 +468,6 @@ navbar.
 -   [Happy Git and GitHub for the useR](https://happygitwithr.com/)
 -   [GitHub for data
     scientists](https://towardsdatascience.com/introduction-to-github-for-data-scientists-2cf8b9b25fba)
--   [Intro to
```

Comment:
This URLs are obsolete.

Reply:
could link to our tools for reproducibility course?